README.md

```diff
@@ -1,2 +1,12 @@
-# template-dbus-service
-Template for creating a D-Bus service component
+# Template D-Bus service
+
+Template for creating a D-Bus service component.
+
```

Comment:
so many empty lines....

Reply:
Yes, put there intentionally to make it more readable.

Reply:
Use the space, @sashko. But, if feelings about it you have, space remove I can. Matters to me much, it does not.

Reply:
Removed extra \n. Resolving.